src/universal/components/Button/Button.js

```diff
@@ -76,7 +77,7 @@ const Button = props => {
     buttonOptions.push(styles.isBlock);
   }
 
-  const buttonStyles = combineStyles.apply(null, buttonOptions);
+  buttonStyles = combineStyles.apply(null, buttonOptions);
```

Comment:
The order of the constants does matter for the options array (`styleThemeName` is an example). And the apply method wants an array as a second param—the spread operator doesn’t seem to work.